components/search_box.js

```diff
@@ -99,7 +98,7 @@ const SearchBox = ({
       {...otherProps}
     />
     {disableButton ? (
-      <DisabledSearchButton title={ariaLabel}>
+      <DisabledSearchButton title={ariaLabel} tabIndex="-1">
```

Comment:
this is to prevent the user from tabbing to a button that doesn't do anything

Reply:
Nice!